src/settings/OwnerSettings.js

```diff
@@ -43,6 +43,16 @@ class OwnerSettings extends React.Component {
     this.connectedControlledVocab = props.stripes.connect(ControlledVocab);
   }
 
+  validate = (item, index, items) => {
+    const { intl: { formatMessage } } = this.props;
+    const label = formatMessage({ id: 'ui-users.owners.singular' });
+    const itemErrors = validate(item, index, items, 'owner', label);
+    if (item.owner === 'Shared' && item.servicePointOwner.length > 0) {
```

Comment:
This works for editing an existing owner named "Shared" but it'll blow up when creating one because the `item.servicePointOwner` array isn't defined yet: 
```
Uncaught TypeError: Cannot read property 'length' of undefined
    at Object.OwnerSettings._this.validate (OwnerSettings.js:50)
    at ControlledVocab.js:314
    ...
```

Reply:
I noticed that, we hope to fix soon.

Reply:
We have made the change.

Reply:
Unfortunately, this does not fix the issue. An empty array evaluates to `true` so, for example, it is not possible to change the description for the `Shared` owner, even when the list of Associated service points is empty.

Reply:
We did the corresponding evaluation of the service point list, it seems to us that this time it does it correctly.